### PR TITLE
Added SDK-Info to HTTP-Request-Header

### PR DESCRIPTION
**Pull request by DerAusHeidelberg**

### CODE REVIEW

**Review by Ryouzanpaku — APPROVED** [review]
LGTM.

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres
 to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.1.2.0][1.1.2.0]
+
+### Added
```
Comment (sixer1182) [inline CHANGELOG.md:10]: + changed:
* Several minor improvements.

**Review by sixer1182 — DISMISSED** [review]
LGTM + Davids comments

File: src/test/java/com/unzer/payment/business/AbstractPaymentTest.java [hunk 2]
```diff
@@ -9,7 +9,10 @@
 import com.unzer.payment.marketplace.MarketplaceCancelBasket;
 import com.unzer.payment.marketplace.MarketplaceCancelBasketItem;
 import com.unzer.payment.marketplace.MarketplaceCharge;
-import com.unzer.payment.paymenttypes.*;
+import com.unzer.payment.paymenttypes.Card;
+import com.unzer.payment.paymenttypes.InvoiceSecured;
+import com.unzer.payment.paymenttypes.SepaDirectDebit;
+import com.unzer.payment.paymenttypes.SepaDirectDebitSecured;
```
[the remark this answers is not in the document]
Comment (DerAusHeidelberg) [reply src/test/java/com/unzer/payment/business/AbstractPaymentTest.java:15]: As you can see there was a "Import all" Statement with "*" so I resolved it to just import the Stuff that is needed.

File: CHANGELOG.md (hunk 1 above)
Comment (DerAusHeidelberg) [reply CHANGELOG.md:10]: Resolved this in another PR.

**Review by Ryouzanpaku — APPROVED** [review]
LGTM.

**Pull request merged**